windows_create_package.bat

```diff
@@ -41,14 +43,12 @@ if !errorlevel! neq 0 exit /b !errorlevel!
 
 :: Prepare self-contained python
 set "dest_dir=C:\opt"
-set "python_version=3.9.13"
+
 call %cd%\windows_prepare_python.bat %dest_dir% %python_version%
 :: Copy whole catalog to dist folder and install dependencies required by LLM pipelines
 xcopy %dest_dir%\python-%python_version%-embed-amd64 dist\windows\ovms\python /E /I /H
 .\dist\windows\ovms\python\python.exe -m pip install "Jinja2==3.1.4" "MarkupSafe==3.0.2"
-if !errorlevel! neq 0 (
-    echo Error copying python into the distribution location. The package will not contain self-contained python.
-)
+if !errorlevel! neq 0 exit /b !errorlevel!
```

Comment:
I'm okay with exiting the script at this point, but I think we could have the `echo` saying that creating package failed because Python environment could not be prepared.